File: LC/lc1/lc1.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
  hash = {};
  for (i = 0; i < nums.length; i++) {
      k = target - nums[i];
      if (k in hash) {
          return [i, hash[k]];
      }
      hash[nums[i]] = i;
  }
  return ans;
};

File: LC/lc1/lc1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashed = {}
        for i in range(len(nums)):
            c = nums[i]
            b = target - c
            if b in hashed:
                return [i, hashed[b]]
            hashed[c] = i
        return [0, 0]